spec/javascripts/components/story/expanded_story/expanded_story_type_spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExpandedStoryType from 'components/story/ExpandedStory/ExpandedStoryType';
+
+describe('<ExpandedStoryEstimate />', () => {
+  it("sets defaultValue as story.storyType in select", () => {
+    const storyTypes = ['feature', 'bug', 'release', 'chore'];
+
+    storyTypes.forEach((type) => {
```

Comment:
Same here, it would be better to loop through the array instead of rendering `ExpandedStoryType` inside the same `it`.